security/advancedtls/advancedtls_test.go

```diff
@@ -384,9 +384,14 @@ func (s) TestClientServerHandshake(t *testing.T) {
 	if err := cs.LoadCerts(); err != nil {
 		t.Fatalf("cs.LoadCerts() failed, err: %v", err)
 	}
-	getRootCAsForClient := func(params *GetRootCAsParams) (*GetRootCAsResults, error) {
+	getRootCAsForClient := func(params *ConnectionInfo) (*RootCertificates, error) {
```

Comment:
Since this was just tests and also not part of the API I'm trying to keep the diff focused on the API.
I don't mind making the changes here either, WDYT?

Reply:
Went on and made the change in both